src/List/ItemList.js

```diff
@@ -39,7 +62,20 @@ const Items = props => {
       ) : (
         <ul>
           {dbItems.map((item, index) => {
-            return <li key={index}>{item.name}</li>;
+            return (
+              <li key={index}>
+                <label>
+                  <input
+                    type="checkbox"
+                    name="isPurchased"
+                    checked={is24Hours(item)}
```

Comment:
I like that we name this function, so that it's clear what the `checked` is getting. Nice!